package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitcore-lib-zen",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Maybe we should increase the MINOR version? 
According to [SemVer2](https://semver.org/) the PATCH version should be bumped if it is a bug. So the questions we need to ask ourselves is: 
1. Is this a bug?
2. Is this backwards compatible
Response to this questions:
1: yes --> PATCH
1:No, 2:yes --> MINOR
1:No, 2:NO --> MAYOR

Reply:
This was changed accordingly to what @cronicc suggested.

Reply:
Ok so we consider it as a Bug then 😄 !
I'm going to start a campaign for everyone to fully understand SemVer versioning (starting by myself 😉 ). 
We have so many applications going on that we need to fully understand how to increase versions.